src/main/java/org/opentripplanner/updater/GraphUpdaterConfigurator.java

```diff
@@ -61,72 +64,75 @@ public static void setupGraph(Graph graph, JsonNode mainConfig) {
      * @param graph the graph that will be modified by these updaters
      * @return a GraphUpdaterManager containing all the created updaters
      */
-    private static GraphUpdaterManager createManagerFromConfig(Graph graph, JsonNode config) {
+    private static List<GraphUpdater> createUpdatersFromConfig(Graph graph, UpdaterConfig config) {
 
-        GraphUpdaterManager updaterManager = new GraphUpdaterManager(graph);
-        for (JsonNode configItem : config.path("updaters")) {
+        List<GraphUpdater> updaters = new ArrayList<>();
+
+        for (UpdaterConfigItem configItem : config.getItems()) {
 
             // For each sub-node, determine which kind of updater is being created.
-            String type = configItem.path("type").asText();
+            String type = configItem.getType();
             GraphUpdater updater = null;
-            if (type != null) {
-                if (type.equals("bike-rental")) {
-                    updater = new BikeRentalUpdater();
-                }
-                else if (type.equals("bike-park")) {
-                    updater = new BikeParkUpdater();
-                }
-                else if (type.equals("stop-time-updater")) {
-                    updater = new PollingStoptimeUpdater();
-                }
-                else if (type.equals("websocket-gtfs-rt-updater")) {
-                    updater = new WebsocketGtfsRealtimeUpdater();
-                }
-                else if (type.equals("real-time-alerts")) {
-                    updater = new GtfsRealtimeAlertsUpdater();
-                }
-                else if (type.equals("example-updater")) {
-                    updater = new ExampleGraphUpdater();
-                }
-                else if (type.equals("example-polling-updater")) {
-                    updater = new ExamplePollingGraphUpdater();
-                }
-                else if (type.equals("winkki-polling-updater")) {
-                    updater = new WinkkiPollingGraphUpdater();
-                }
-                else if (type.equals("siri-et-updater")) {
-                    updater = new SiriETUpdater();
-                }
-                else if (type.equals("siri-vm-updater")) {
-                    updater = new SiriVMUpdater();
+
+            try {
+                if (type != null) {
+                    switch (type) {
+                        case "bike-rental":
+                            updater = new BikeRentalUpdater();
+                            ((BikeRentalUpdater) updater).configure(graph, configItem);
```

Comment:
Comment: I guess you are gone change this later to just: 
```
updater = new BikeRentalUpdater(configItem);
```
And then inject the graph later.